src/components/tab-bar/tab-bar.tsx

```
@@ -119,25 +121,25 @@ export class TabBar {
                     <div class="scroll-fade right" />
                     <div class="scroll-button right">
-                        <limel-icon-button
-                            icon="angle_right"
-                            elevated={true}
+                        <button
                             tabindex="-1"
                             aria-hidden="true"
                             disabled={!this.canScrollRight}
                             onClick={this.handleRightScrollClick}
-                        />
+                        >
```

Biome (1.9.4)

[error] Disallow aria-hidden="true" from being set on focusable elements.

aria-hidden should not be set to true on focusable elements because this can lead to confusing behavior for screen reader users.
Unsafe fix: Remove the aria-hidden attribute from the element.


(lint/a11y/noAriaHiddenOnFocusable)